docs/sources/overview/_index.md

```diff
@@ -2,31 +2,79 @@
 title: Overview
 weight: 150
 ---
-# Overview of Loki
+# Overview
 
-Grafana Loki is a set of components that can be composed into a fully featured
-logging stack.
+Grafana Loki is a log aggregation tool,
+and it is the core of a fully-featured logging stack.
 
-Unlike other logging systems, Loki is built around the idea of only indexing
-labels for logs and leaving the original log message unindexed. This means
-that Loki is cheaper to operate and can be orders of magnitude more efficient.
+Loki is a key-value datastore optimized for efficiently holding log data.
+The efficient indexing of log data
+distinguishes Loki from other logging systems.
+Unlike other logging systems, a Loki index is built from labels,
+leaving the original log message unindexed.
 
-For a more detailed version of this same document, please read
-[Architecture](../architecture/).
+![Loki overview](loki-overview-1.png)
 
-## Multi Tenancy
+An agent (also called a client) acquires logs,
+turns the logs into streams,
+and pushes the streams to Loki through an HTTP API.
+The Promtail agent is designed for Loki installations,
+but many other [Agents](../clients/) seamlessly integrate with Loki.
 
-Loki supports multi-tenancy so that data between tenants is completely
-separated. Multi-tenancy is achieved through a tenant ID (which is represented
-as an alphanumeric string). When multi-tenancy mode is disabled, all requests
-are internally given a tenant ID of "fake".
+![Loki agent interaction](loki-overview-2.png)
+
+Loki indexes streams.
+Each stream identifies a set of logs associated with a unique set of labels.
+A quality identification and specification of the set of labels
+is key to the creation of an index that is both compact
+and allows for efficient query execution.
```

Comment:
I've read this a few times over and am having some difficulty understanding this sentence. Can we simplify?
```suggestion
A quality set of labels is key to the creation of an index that is both compact
and allows for efficient query execution.
```